python/lsst/ts/mteec/mteec_csc.py

```diff
@@ -111,7 +113,9 @@ async def configure(self, config):
 
     async def implement_simulation_mode(self, simulation_mode):
         if simulation_mode not in (0, 1):
-            raise salobj.ExpectedError(f"Simulation_mode={simulation_mode} must be 0 or 1")
+            raise salobj.ExpectedError(
+                f"Simulation_mode={simulation_mode} must be 0 or 1"
+            )
```

Comment:
I suggest deleting this method. salobj will check the value for you because you set class variable valid_simulation_modes

Reply:
Done